src/main/webapp/app/entities/file-upload-assessment/file-upload-assessment.service.ts

```diff
@@ -0,0 +1,43 @@
+import { Injectable } from '@angular/core';
+import { SERVER_API_URL } from 'app/app.constants';
+import { HttpClient, HttpResponse } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { Result } from 'app/entities/result';
+import { Feedback } from 'app/entities/feedback';
+import { ComplaintResponse } from 'app/entities/complaint-response';
+
+export type EntityResponseType = HttpResponse<Result>;
+
+@Injectable({
+    providedIn: 'root',
+})
+export class FileUploadAssessmentsService {
+    private resourceUrl = SERVER_API_URL + 'api';
+
+    constructor(private http: HttpClient) {}
+
+    saveAssessment(feedbacks: Feedback[], submissionId: number, submit = false): Observable<Result> {
+        let url = `${this.resourceUrl}/file-upload-submissions/${submissionId}/feedback`;
+        if (submit) {
```

Comment:
Nit: It would be nice to write a small util for this that appends params to the URL.

Reply:
Added `buildUrlWithParams` to `global.utils` file